lib/license_finder/package_managers/rebar.rb

```diff
@@ -0,0 +1,60 @@
+require 'json'
+
+module LicenseFinder
+  class Rebar < PackageManager
+    def initialize(options={})
+      super
+      @command = options[:rebar_command] || "rebar"
+      @deps_path = options[:rebar_deps_dir] || "deps"
```

Comment:
If you change this to 
```ruby
@deps_path = Pathname(options[:rebar_deps_dir] || "deps")
```
then below you can write
```ruby
@deps_path.join(dep["name"])
```
which is slightly shorter... we try to use Pathname if we can.